### DISCUSSION TITLE
Mapbox actively promotes React Native for AR

### DISCUSSION THREAD

**Discussion by Gnative** [rnmapbox/maps#2599] [General]
Found this interesting that Mapbox is promoting React Native for AR on their features page - https://www.mapbox.com/augmented-reality 

It links to this repo https://github.com/mapbox/react-native-mapbox-ar  with the last commit on Jul 2, 2018  ..

**Comment by SammyAlvarado** [comment]
Is this AR sdk no longer being maintained? I tried looking for a good start guide but was not able to fine one. Was wondering if anyone had found a good resources?
